lib/app.jsx

```diff
@@ -99,6 +99,11 @@ const isElectron = ( () => {
 	return () => foundElectron;
 } )();
 
+const isElectronMac = () => {
+	return isElectron() &&
+		get( window, 'process.platform', '' ) === 'darwin';
+}
```

Comment:
Cheers, I didn't realize that `process` was provided by Electron. Also updated to use `matchesProperty` 👍